EXP-Client/App.js

```diff
@@ -31,10 +32,28 @@ export default function App() {
     { name: 'Task 4', deadline: 'April 22, 2023', completed: true }
   ];
 
+  const [todayList, setTodayList] = React.useState([]);
+
+  let instance = axios.create({
+    baseURL: "https://explife-backend.fly.dev"
+  });
+
+  instance
+    .get("/lists/:id")
+    .then((res) => {
+      setTodayList(res.data)
+    })
+    .catch((error) => {
+      console.log(error)
+    })
+
   return (
     <Provider theme={MD3LightTheme}>
       <View style={styles.nav}>
-        <Nav Todo={<Todo tasks={tasks} />} />
+        <Nav
+          Todo={<Todo tasks={tasks} />}
+          Progress={<ArchivedDay tasks={tasks} />}

```

Comment:
Will Progress lead to a single Archived Day or a list of Archived days to choose from? I think we decided it should go to a list from where it should go to a single day

Reply:
Think it's up to implementation but this is out of the scope of this PR ;-;